libs/partners/ai21/README.md

```diff
@@ -28,13 +28,40 @@ Then initialize
 from langchain_core.messages import HumanMessage
 from langchain_ai21.chat_models import ChatAI21
 
-chat = ChatAI21(model="jamba-instruct-preview")
+chat = ChatAI21(model="jamba-instruct")
```

Comment:
@Josephasafg resolved a merge conflict due to this change: https://github.com/langchain-ai/langchain/pull/23353/files

Hope that's correct.

Reply:
LGTM :)